### DISCUSSION TITLE
[@types/node] v18.17.0: Missing new `path` property on `fs.Dirent` class

### DISCUSSION THREAD

**Discussion by djvergel** [DefinitelyTyped/DefinitelyTyped#66402] [Issues with a @types package] [answered]
A new property (`path`) on the `fs.Dirent` class was added in node 18.17.0 and is missing from [@types/node] v18

https://nodejs.org/dist/latest-v18.x/docs/api/fs.html#direntpath

Here's the property in the node.js source: https://github.com/nodejs/node/blob/v18.17.0/lib/internal/fs/utils.js#L165

Can that be added?

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
fixing in #66405
